app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,99 @@
+body {
+  margin: 0;
+}
+
+header {
+  width: 100%;
+  position: fixed;
+  #logoMain {
+    fill: #190fd1;
+  }
+}
+
+.container {
+  max-width: 1500px;
+  padding: 0 80px;
+  margin: 0 auto;
+  background-color: #FFFFFF;
+}
+
+.br {
+  border-radius: 0 0 20px 20px;
+}
+
+#logoMain {
+  padding: 15px 0;
+  width: 114px;
+  height: 41px;
+}
+
+.menu {
+  display: flex;
+  a {
+    padding: 20px 30px;
+    color: #000;
+    font-weight: 700;
+    &:hover {
+      color: #190fd1;
+    }
+  }
+}
+
+.top-bar {
+  padding: 30px 0;
+  transition: all 0.5s ease;
+  display: flex;
```

Comment:
styles of blocks, positioning and everything that can affect other blocks are styled at the top, and only then come specific styles such as animation

Reply:
fixed